tests/inference_runner.py

```diff
@@ -57,6 +61,14 @@ def print_context(ctx, ind=""):
     print([solver.assertions_errors[x] for x in solver.unsat_core()])
 else:
     model = solver.optimize.model()
-    print_context(context)
+    context.generate_typed_ast(model, solver)
+
+    # uncomment this to write typed source into a file
```

Comment:
Not saying this needs to be done now, but at some point it might be nice to have a main method to call that takes parameters.